### ISSUE TITLE
SBL§6.3.8 A Paper Presented at a Professional Society (p. 95)

### ISSUE THREAD

**Issue by spencerclrk** [dcpurton/biblatex-sbl#41]
#### Capitalization

**Footnote should be lowercase (below is wrong)**
>1. Elizabeth Schrader, “Was Martha of Bethany added to the Fourth Gospel in the Second Century?” (Paper presented at the Annual Meeting of the Society of Biblical Literature, Atlanta, 23 November 2015).

**Bibliography should be capitalized (below is correct)**
> Schrader, Elizabeth. “Was Martha of Bethany added to the Fourth Gospel in the Second Century?” Paper presented at the Annual Meeting of the Society of Biblical Literature. Atlanta, 23 November 2015.

I'm guessing the `\autocap` case tracking isn't working just yet.

---

#### Regarding `note`
 I assume you're leaving this “as is” for flexibility purposes with `misc`. An alternative I've tried before is writing “paper presented at the” into the .cbx/.bbx files and create separate fields for `eventtitle` and `organization` (see BibLaTeX documentation).

The question is, I guess, whether “Paper at a Conference” is significant enough to have its own rules apart from `unpublished` or `misc`. Personally I use this all the time but have rarely needed a generic `unpublished` category. What do others think?

**Comment by dcpurton** [comment]
Ah! Interesting. I've changed how I do this one a couple of times, so it's definitely worth some discussion.

First, `\autocap` is working… The title in the above example ends with a question mark, which causes biblatex's punctuation tracking to capitalise the next *paper*, since it thinks a new sentence is starting. This is probably a pretty rare occurrence outside of this particular example, although it may crop up elsewhere. Let's think about the best way to do this entry type before thinking about that problem.

Initially I was using `@inproceedings` for this type, and I did have “paper presented at” in the driver. But then I switch to `@unpublished` because it seemed like `@inproceedings` was meant to be for published conference proceedings, whereas §6.3.8 seems to be an unpublished paper. Are my assumptions correct?

I also did use the `note` field because of flexibility. I might be mistaken, but it seems to me hard to predict how the `eventtitle` should relate to "paper presented at the" and `organization` (joining words might not be consistent…).

There are (at least) two potential useful options.
  1. Create a new `@conferencepaper` `entrytype`
  2. Create a conferencepaper `entrysubtype` for `@unpublished` entries

Here's a few options. What do you like? Option 2.c gives the nicest database entry, but the least flexibilty. Option 2.b might be the best compromise. All of the new options will solve your case problem above.

## 1. Current option
```
@unpublished{schrader:2015,
  author = {Schrader, Elizabeth},
  title = {Was Martha of Bethany added to the Fourth Gospel in the Second Century?},
  note = {\autocap{p}aper presented at the Annual Meeting of the Society of Biblical Literature},
  location = {Atlanta},
  date = {2015/11/23}
}
```

## 2. Create a new `@conferencepaper` main `entrytype`

### Option a
```
@conferencepaper{schrader:2015,
  author = {Schrader, Elizabeth},
  title = {Was Martha of Bethany added to the Fourth Gospel in the Second Century?},
  eventtitle = {the Annual Meeting of the Society of Biblical Literature},
  venue = {Atlanta},
  eventdate = {2015/11/23}
}
```

Driver would need something like:

```TeX
\printtext{paper presented at}%
\setunit{\space}%
\printfield{eventtitle}%
\newunit
\printfield{venue}%
\newunit
\printfield{eventdate}%
```

### Option b
```
@conferencepaper{schrader:2015,
  author = {Schrader, Elizabeth},
  title = {Was Martha of Bethany added to the Fourth Gospel in the Second Century?},
  eventtitle = {the Annual Meeting of},
  organization = {the Society of Biblical Literature},
  venue = {Atlanta},
  eventdate = {2015/11/23}
}
```

Driver would need somethig like:

```TeX
\printtext{paper presented at}%
\setunit{\space}%
\printfield{eventtitle}%
\setunit{\space}%
\printfield{organization}%
\newunit
\printfield{venue}%
\newunit
\printfield{eventdate}%
```

### Option c
```
@conferencepaper{schrader:2015,
  author = {Schrader, Elizabeth},
  title = {Was Martha of Bethany added to the Fourth Gospel in the Second Century?},
  eventtitle = {Annual Meeting},
  organization = {Society of Biblical Literature},
  venue = {Atlanta},
  eventdate = {2015/11/23}
}
```

Driver would need something like:

```TeX
\printtext{paper presented at the}%
\setunit{\space}%
\printfield{eventtitle}%
\setunit{\space}%
\printtext{of the}%
\setunit{\space}%
\printfield{organization}%
\newunit
\printfield{venue}%
\newunit
\printfield{eventdate}%
```

## 3. Use `@unpublished` with a conferencepaper `entrysubtype`

Repeat previous three options, but use `entrysubtype` to create a special kind of unpublished entry.

**Comment by spencerclrk** [comment]
Yeah my understanding of `@inproceedings` is that it is for published proceedings. There really isn't a good native field for a conference paper

For most uses **2c** is my personal preference. But I do realize that there are other kinds of conferences that could cause issues:

> John Smith, “What's in a Name?” (Paper presented at the Layman Scholars Conference **of the** Prestigious University, Chicago, 23 November 2015).

```
@conferencepaper{smith:2015,%
  author = {Smith, John},
  title = {What's in a Name?},
  eventtitle = {Layman Scholars Conference},
  organization = {Prestigious University},
  venue = {Chicago},
  eventdate = {2015/11/23}
}
```

“Of the” could work, but maybe “at the Prestigious University” or something else would be more appropriate. 

Do you have any other possible examples of events, being “academic enough” for citation, that might have issues with type **2c**?

**Comment by dcpurton** [comment]
Hmm - I only have an MDiv, so I'm not qualified to comment on what kind of conferences there are ;).

I could make the "of the" printing dependent on the presence of an `organization` field. So if you wanted something else, you could just put everything in `eventtitle`.

```
@conferencepaper{smith:2015,
  author = {Smith, John},
  title = {What's in a Name?},
  eventtitle = {Layman Scholars Conference at the Prestigious University},
  venue = {Chicago},
  eventdate = {2015/11/23}
}
```

**Comment by spencerclrk** [comment]
You're further along than me ;D

And that solution seems reasonable. Also if you added support for `institution` you could differentiate between the two. From `biblatex.pdf`

- `organization` —  “the organization(s) that published a `@manual` or an `@online` resource, or spon-
sored a conference.”
- `institution` — “The name of a university or some other institution, depending on the entry type.”

**Comment by dcpurton** [comment]
ok. I'll implement this. I won't do `institution` for the moment. I think it might end up being a little unintuitive to have joining words randomly with the use of a different field.

**Comment by Nhapsie** [comment]
I am doing a PhD, but know far less LaTeX than both of you. For me the only thing I use is SBL conference papers. By looking at the descriptive side of the entry, `inproceedings` was not only a wrong entry type, but mainly non-intuitive. `Unpublished` is also not right, because it may refer to a non-published book, not a conference paper. So `conferencepaper` is the best option, you can't go wrong. Having said that, I would go for 2A for the simple reason that there is only one kind of entry used for both annual and regionsal SBL meetings.

**Comment by dcpurton** [comment]
On reflection, I'm inclined to agree with @Nhapsie on this one. Have a look at the latest dev branch and see the test document for usage. It works correctly with your example.

**Comment by Nhapsie** [comment]
Good, I remember there’s also SBL International Meetings, but it uses the same format.

**Comment by dcpurton** [comment]
fixed in v0.7
